pkg/scheduler/framework/plugins/noderesources/fit.go

```diff
@@ -189,6 +190,18 @@ func getPreFilterState(cycleState *framework.CycleState) (*preFilterState, error
 	return s, nil
 }
 
+// EventsToRegister returns the possible events that may make a Pod
+// failed by this plugin schedulable.
+// NOTE: if in-place-update (KEP 1287) gets implemented, PodUpdate will have
+// has a new semantics, by then we need to register PodUpdate b/c an assigned
+// Pod may occupy less resources.
```

Comment:
```suggestion
// NOTE: if in-place-update (KEP 1287) gets implemented, then PodUpdate events should be registered
// for this plugin since a pod update may free up resources that make other pods schedulable.
```

Reply:
updated.